cobc/ChangeLog

```diff
@@ -1,3 +1,6 @@
+2025-10-17  Roger Bowler <rbowler@snipix.net>
```

Comment:
first line should stay empty (the diff works better in several clients then) - but no need for an extra commit only for that